### ISSUE TITLE
Support file uploads within models

### ISSUE THREAD

**Issue by erkie** [erkie/ApiModel#4]
I'm thinking something like this:

``` swift
import APIModel
import Realm

class User: Object, ApiTransformable {
     dynamic var name = ""
     dynamic var avatar = ""

     func JSONDictionary() -> [String:Anyobject] {
         var avatarPath = NSBundle.mainBundle().pathForResource("avatar", ofType:"jpg")

         return [
              "name": name,
              "avatar": NSData(contentsOfFile: avatarPath) 
         ]
     }
}
```

**Comment by erkie** [comment]
I'll work on this next.

**Comment by erkie** [comment]
Closed by `0.10.0` release.
